events/wc_culture_conversion_events.txt

```diff
@@ -0,0 +1,93 @@
+
+# ###########################################
+# # Warcraft culture conversion events   	#
+# ###########################################
+
+namespace = WCMLT
+
+# ###########################################
+# # Province conversions                    #
+# ###########################################
+
+# Human province in Kalimdor converts to Theramorean
+province_event = {
+	id = WCMLT.1
+	desc = EVTDESC_WCMLT_1
+	picture = GFX_evt_lordaeron_coast
+
+	trigger = {
+		should_evolve_cultures_trigger = yes
+		culture_group = human_group
+		NOT = { culture = western }
+		OR = {
+			region = world_kalimdor_barrens		 
+			region = world_kalimdor_dustwallow
+			region = island_theramore
+			region = island_alcaz
+		}
+		any_province_holding = {
+			NOT = { holding_type = nomad }
+		}
+	}
+	
+	mean_time_to_happen = {
+		months = 480
+		modifier = {
+			factor = 0.3
+			OR = {
+				has_game_rule = {
+					name = culture_conversion
+					value = faster_melting_pots
+				}
+				has_game_rule = {
+					name = culture_conversion
+					value = faster_melting_pots_and_slower_normal
+				}
+			}
+		}
+	}
+
+	option = {
+		name = EVTOPTA_WCMLT_1
+		culture = western
+	}
+}
+
+# Orcish province in Kalimdor converts to Durotic
+province_event = {
+	id = WCMLT.2
+	desc = EVTDESC_WCMLT_2
+	picture = GFX_evt_durotar
+
+	trigger = {
+		should_evolve_cultures_trigger = yes
+		culture_group = orc_group
+		NOT = { culture = durotaric }
+		region = world_kalimdor_barrens
```

Comment:
This one is more or less up to debate, but limited it to close proximity of Durotar due to the multi-clan nature of the orcish nation, while independent clans can still retain their identity outside of it.

Reply:
The problem is that Barrens include a lot of lands. Even the main base of the Warsong clan in the Northern Barrens. The clan was part of the New Horde from the very beginning and it will be very strange that it will begin to lose its own culture.

Reply:
Aye.

Reply:
Okie-dokie, I'll implement it.

Reply:
What do you think about making it convert orcs in Azshara as well?